Session/Calls/Call Management/SessionCall.swift

```diff
@@ -277,7 +278,8 @@ public final class SessionCall: CurrentCallProtocol, WebRTCSessionDelegate {
         let duration: TimeInterval = self.duration
         let hasStartedConnecting: Bool = self.hasStartedConnecting
         
-        Storage.shared.writeAsync { db in
+        // Don't know whether there will be an impact if we don't write async
```

Comment:
Generally we should `writeAsync` unless we have a specific reason to write synchronously as it'll block the thread until the write operation is done (which can cause visual issues if running on the main thread even if it's really quick)

Reply:
The only reason I changed this to user write is when the call is handling in background, the info message will need to be updated before the database is suspended. I can move the suspended logic here.

Reply:
Yea no worries, I don't really have a preference - just wanted to make sure there was a reason, the worst case scenario would just be that this happens to run on the main thread and the user is navigating the app when this code gets hit (and the UI gets a bit jittery)